libs/common/src/assets/i18n/fr.json

```
@@ -1267,6 +1267,7 @@
 	"retrieval-agents.workflow.node-types.external.form.prompt.label": "Extra prompt",
 	"retrieval-agents.workflow.node-types.external.form.url": "URL à appeler",
 	"retrieval-agents.workflow.node-types.external.title": "Appel externe",
+	"retrieval-agents.workflow.node-types.generate.config.default-prompt": "Générez du texte / JSON / image à l’aide du prompt par défaut",
```

I guess it's more that, no?
```suggestion
	"retrieval-agents.workflow.node-types.generate.config.default-prompt": "Générer du texte / JSON / image à l’aide du prompt par défaut",
```